packages/prometheus/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@hono/prometheus",
+  "version": "0.1.0",
+  "description": "Prometheus metrics middleware for Hono",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "test": "vitest --run",
+    "build": "tsup ./src/index.ts --format esm,cjs --dts",
+    "publint": "publint",
+    "release": "yarn build && yarn test && yarn publint && yarn publish"
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.mts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    }
+  },
+  "license": "MIT",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org",
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/honojs/middleware.git"
+  },
+  "homepage": "https://github.com/honojs/middleware",
+  "peerDependencies": {
+    "hono": "^3.9.2",
```

Comment:
Since `3.11.6` or lower is a security risk, specify the latest version, `3.12.0` or higher.

Reply:
I used another package as a base which turned out wasn't the best choice :) Thanks, I've updated this to 3.12.0